homeassistant/components/hue/scene.py

```diff
@@ -97,6 +99,11 @@ def __init__(
         self.resource = resource
         self.controller = controller
         self.group = self.controller.get_group(self.resource.id)
+        # we create a virtual service/device for Hue zones/rooms
+        # so we have a parent for grouped lights and scenes
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, self.group.id)},
+        )
```

Comment:
Why are we removing the rest of the info?

Reply:
The device is now precreated elsewhere, so the duplicate code is removed.